src/std/fs.ab

```diff
@@ -1,55 +1,66 @@
+import * from "std/env"
+
+/// Check if directory exists
 pub fun dir_exist(path) {
     $[ -d "{path}" ]$ failed {
         return false
     }
     return true
 }
 
+/// Check if file exists
 pub fun file_exist(path) {
     $[ -f "{path}" ]$ failed {
         return false
     }
     return true
 }
 
+/// Get the file content
 pub fun file_read(path) {
     return $< "{path}"$?
 }
 
+/// Write the content to the file, doesn't check if the file exist
 pub fun file_write(path, content) {
     return $echo "{content}" > "{path}"$?
 }
 
+/// Append the content to the file, doesn't check if the file exist
 pub fun file_append(path, content) {
     return $echo "{content}" >> "{path}"$?
 }
 
+/// Create a symbolic link, if the file doens't exist return a boolean and print a message
 pub fun create_symbolic_link(origin: Text, destination: Text): Bool {
     if file_exist(origin) {
         unsafe $ln -s "{origin}" "{destination}"$
         return true
     }
 
-    echo "The file {origin} doesn't exist!"
+    error("The file {origin} doesn't exist!")
```

Comment:
Wondering if we want to `exit` the program when it fails, after all the program is crashing otherwise.

Reply:
I think just this function can be fail.
if you exit user lost control and cannot catch error

Reply:
We can show the warning and exit, like most of the scripting languages does when a file doens't exist

Reply:
if we display warnings in functions, this should be able to be deactivated by user.

Reply:
It is something we need to decide in another ticket.

Good point

Reply:
```
pub fun create_symbolic_link(path: Text): Bool {
  silent $ln -s "{origin}" "{destination}"$ failed {
          return false
  }    
  return true
}
```